src/components/sidebar/sidebar.tsx

```diff
@@ -58,70 +53,60 @@ interface SidebarProps {
     const sidebarRef = useRef<HTMLDivElement>(null);
-    const selectedTab = useSidebarViewStore(erPaMinOversikt ? ListevisningType.minOversikt : ListevisningType.enhetensOversikt)
+    const selectedTab = useSidebarViewStore(
+        erPaMinOversikt ? ListevisningType.minOversikt : ListevisningType.enhetensOversikt
+    );
     const selectedTabData = finnTab(selectedTab.selectedTab, sidebar);
-    const mineFilterState = useSelector((state: AppState) => state.mineFilter);
     const dispatch = useDispatch();
     const windowWidth = useWindowWidth();
     const isSidebarHidden = useSidebarViewStore(props.filtergruppe).isSidebarHidden;
 
     const tabFocus = () => {
         if (erPaMinOversikt) {
-            if (selectedTabData.type === "STATUS") return 0;
-            else if (selectedTabData.type === "MINE_FILTER") return 1;
-            else if (selectedTabData.type === "FILTER") return 2;
-
+            if (selectedTabData.type === 'STATUS') return 0;
+            else if (selectedTabData.type === 'MINE_FILTER') return 1;
+            else if (selectedTabData.type === 'FILTER') return 2;
         } else {
-            if (selectedTabData.type === "STATUS") return 0;
-            else if (selectedTabData.type === "MINE_FILTER") return 1;
-            else if (selectedTabData.type === "VEILEDERGRUPPER") return 2;
-            else if (selectedTabData.type === "FILTER") return 3;
+            if (selectedTabData.type === 'STATUS') return 0;
+            else if (selectedTabData.type === 'MINE_FILTER') return 1;
+            else if (selectedTabData.type === 'VEILEDERGRUPPER') return 2;
+            else if (selectedTabData.type === 'FILTER') return 3;
         }
         return 0;
-    }
+    };
 
     let tabFoc = tabFocus();
 
     const keyCode = (e) => e.which || e.keyCode;
 
-    useEffect(() => {
-        const nyttLagretFilter = mineFilterState.handlingType === HandlingsType.NYTT && mineFilterState.status === STATUS.OK;
-        const oppdatertLagretFilter = mineFilterState.handlingType === HandlingsType.REDIGERE && mineFilterState.status === STATUS.OK;
-
-        if (nyttLagretFilter || oppdatertLagretFilter) {
-            dispatch({
-                type: SIDEBAR_TAB_ENDRET,
-                selectedTab: SidebarTabInfo.MINE_FILTER,
-                name: erPaMinOversikt ? ListevisningType.minOversikt : ListevisningType.enhetensOversikt
-            })
-        }
-    }, [dispatch, erPaMinOversikt, mineFilterState.handlingType, mineFilterState.status])
-
     function finnTab(viewType: SidebarTabType, tabs: Sidebar[]): Sidebar {
-        return tabs.find(t => t.type === viewType) as Sidebar;
+        return tabs.find((t) => t.type === viewType) as Sidebar;
     }
 
-    const mapTabTilView = (tab: Sidebar, isSelected: boolean,) => {
+    const mapTabTilView = (tab: Sidebar, isSelected: boolean, key: number) => {
+        const ariaFaneTekst = isSidebarHidden ? 'Tab lukket.' : 'Tab åpen.';
```

Comment:
Kanskje kalle det "fane" istedenfor "tab"?

Reply:
Var enig med deg før jeg testet, da min skjermleser refererer til det som en tab. I tillegg så er jeg litt redd for å endre det siden David har sett på det uten å kommentere det.

Reply:
Ok, da er jeg enig!